skimage/measure/fit.py

```diff
@@ -573,6 +573,10 @@ def ransac(data, model_class, min_samples, residual_threshold,
         where the probability (confidence) is typically set to a high value
         such as 0.99, and e is the current fraction of inliers w.r.t. the
         total number of samples.
+    exceptions : exception class or tuple of exception classes
```

Comment:
Is it wise to have anything in there by default?  This looks like a recipe for disaster.